CONTRIBUTING.md

```diff
@@ -136,6 +136,8 @@ effect on highlighting. We will work on improving highlighting in the near futur
 @keyword
 @keyword.function
 @keyword.operator (for operators that are English words, e.g. `and`, `or`)
+@keyword.return
+@keyword.yield
```

Comment:
I would be more inclined to have those two as only one.

For me, even if those represent semantically two different things, they actually mean "return control to the caller".

Thus, I'd only have `keyword.return`.

Reply:
This makes sense, thank you